clr-k8s-examples/create_stack.sh

```diff
@@ -108,6 +109,7 @@ function cluster_init() {
 	#Ensure single node k8s works
 	if [ "$(kubectl get nodes | wc -l)" -eq 2 ]; then
 		kubectl taint nodes --all node-role.kubernetes.io/master-
+		mode="standalone"
```

Comment:
Would this change work when a user does not use `create_stack.sh all` and creates the cluster one piece at a time? For eg: if i do `create_stack.sh init`, `create_stack.sh cni` and `create_stack.sh storage`, i might end up a hung cluster, correct? Would it help if we do the check of whether we have a single node or more right before we apply rook and then choose which set of manifests to apply?